### PR TITLE
Add ability to deploy to quay on release

### PR DESCRIPTION
**Pull request by shawn-hurley**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+script:
+  - "make build test"
+
+before_deploy:
+  - make image "${IMAGE}"
+  - docker login quay.io -u "${DOCKER_USER}" -p "${DOCKER_PASS}"
+  - docker tag "$IMAGE" "${IMAGE}:latest"
+  - docker tag "$IMAGE" "${IMAGE}:${TRAVIS_TAG}"
+
+depoloy:
```
Comment (lilic) [inline .travis.yml]: think this might be a typo: deploy?

**Review by lilic — CHANGES_REQUESTED** [review]
When does deploy get triggered? On every PR merge?

File: .travis.yml (hunk 1 above)
Comment (shawn-hurley) [reply .travis.yml]: 🤦‍♂️

**Review by pmorie — COMMENTED** [review]
We need to make sure the travis robot has access to the quay repository, right?

**Comment by shawn-hurley** [comment]
@pmorie I believe that I have added the access and the environment variables to travis

**Pull request closed without merging**